### PR TITLE
Add sms, snapchat and messenger share feature for Android

### PR DESCRIPTION
**Pull request by muhzi4u**
I have added Sms, Snapchat and messenger share feature for Android. But I don't know about iOS.

### CODE REVIEW

File: android/src/main/java/cl/json/social/MessengerShare.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public void open(ReadableMap options) throws ActivityNotFoundException {
+        super.open(options);
+        //  extra params here
+        this.openIntentChooser();
+    }
+
+    @Override
+    protected String getPackage() {
+        return PACKAGE;
+    }
```
Comment (mikehardy) [inline android/src/main/java/cl/json/social/MessengerShare.java]: It seems that the getPackage / getDefaultWebLink / getPlayStoreLink implementations here are all repeated and add no value.

Would it be simpler to add default implementations to SingleShareIntent, then in these objects simply set SingleShareIntent.appStoreURL and SingleShareIntent.playStoreURL correctly?

**Comment by kranthijulakantiwork** [comment]
Why is this feature not getting merged?

**Comment by jgcmarins** [comment]
Friendly reminder: @muhzi4u ping \o/

**Comment by muhzi4u** [comment]
@jgcmarins I don't have experience with ios. I will update the README file within one day

**Comment by jgcmarins** [comment]
> @jgcmarins I don't have experience with ios.

That's ok, don't worry.

> I will update the README file within one day

That's awesome!

**Comment by muhzi4u** [comment]
@jgcmarins Could you please merge the branch?

**Pull request merged**